### ISSUE TITLE
Test fail if cdflib is absent

### ISSUE THREAD

**Issue by shreyasbapat** [heliopython/heliopy#701]
Two test modules are failing to load when cdflib is not installed

```
shreyas@canopus ~/L/heliopy> pytest -vv
==================================================================================== test session starts =====================================================================================
platform linux -- Python 3.7.2, pytest-3.8.0, py-1.6.0, pluggy-0.7.1 -- /home/shreyas/Softwares/anaconda3/bin/python
cachedir: .pytest_cache
Matplotlib: 2.2.3
Freetype: 2.9.1
rootdir: /home/shreyas/Local Forks/heliopy, inifile: pytest.ini
plugins: remotedata-0.3.0, openfiles-0.3.0, mpl-0.10, doctestplus-0.1.3, cov-2.5.1, arraydiff-0.2
collected 1 item / 2 errors                                                                                                                                                                  

=========================================================================================== ERRORS ===========================================================================================
______________________________________________________________________ ERROR collecting heliopy/data/test/test_data.py _______________________________________________________________________
ImportError while importing test module '/home/shreyas/Local Forks/heliopy/heliopy/data/test/test_data.py'.
Hint: make sure your test modules/packages have valid Python names.
Traceback:
../../Softwares/anaconda3/lib/python3.7/site-packages/_pytest/python.py:481: in _importtestmodule
    mod = self.fspath.pyimport(ensuresyspath=importmode)
../../Softwares/anaconda3/lib/python3.7/site-packages/py/_path/local.py:668: in pyimport
    __import__(modname)
../../Softwares/anaconda3/lib/python3.7/site-packages/_pytest/assertion/rewrite.py:296: in load_module
    six.exec_(co, mod.__dict__)
heliopy/data/test/test_data.py:1: in <module>
    import heliopy.data.wind as wind
heliopy/data/wind.py:13: in <module>
    import cdflib
E   ModuleNotFoundError: No module named 'cdflib'
_____________________________________________________________________ ERROR collecting heliopy/spice/test/test_spice.py ______________________________________________________________________
ImportError while importing test module '/home/shreyas/Local Forks/heliopy/heliopy/spice/test/test_spice.py'.
Hint: make sure your test modules/packages have valid Python names.
Traceback:
../../Softwares/anaconda3/lib/python3.7/site-packages/_pytest/python.py:481: in _importtestmodule
    mod = self.fspath.pyimport(ensuresyspath=importmode)
../../Softwares/anaconda3/lib/python3.7/site-packages/py/_path/local.py:668: in pyimport
    __import__(modname)
heliopy/spice/__init__.py:3: in <module>
    import heliopy.data.spice as dataspice
heliopy/data/spice.py:19: in <module>
    import heliopy.data.util as util
heliopy/data/util.py:20: in <module>
    import cdflib
E   ModuleNotFoundError: No module named 'cdflib'
====================================================================================== warnings summary ======================================================================================
/home/shreyas/Softwares/anaconda3/lib/python3.7/site-packages/socks.py:58: DeprecationWarning: Using or importing the ABCs from 'collections' instead of from 'collections.abc' is deprecated, and in 3.8 it will stop working
  from collections import Callable

/home/shreyas/Softwares/anaconda3/lib/python3.7/site-packages/sqlalchemy/engine/result.py:182: DeprecationWarning: Using or importing the ABCs from 'collections' instead of from 'collections.abc' is deprecated, and in 3.8 it will stop working
  from collections import Sequence

-- Docs: https://docs.pytest.org/en/latest/warnings.html
!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!! Interrupted: 2 errors during collection !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
============================================================================ 2 warnings, 2 error in 15.83 seconds ============================================================================
```

**Comment by dstansby** [comment]
This is because `cdflib` isn't a requirement for installation, but is a requirement for running the tests and using the `heliopy.data` module. It's defined here in setup.py:
https://github.com/heliopython/heliopy/blob/fd5cf907dbf796a6eb8aa5604145da2c29064db6/setup.py#L36-L38

Is there somewhere else this needs to be made a requirement for testing?

**Comment by shreyasbapat** [comment]
This issue comes because `cdflib` is not a package on Debian. We can package it too. But  what I can see in `test_data.py` is that you have already handled this exception and skipped those tests if cdflib us not installed.

https://github.com/heliopython/heliopy/blob/master/heliopy/data/test/test_data.py#L28-L32

I think #702 must fix this issue. Please have a look once

**Comment by dstansby** [comment]
I think this is now fixed by #705, could you take a look and let me know if you agree?

**Comment by shreyasbapat** [comment]
Right! Thanks a lot for the change @dstansby  :) 

I will try to package it.
